### PR TITLE
scss beautification

### PR DESCRIPTION
**Pull request by NikolaiYurchenko**
What: 
scss beautification of web/satellite
Why:
fix incorrect spacing in scss
Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/account/EditProfilePopup.vue [hunk 1]
```diff
@@ -122,138 +122,142 @@
...
-			&:hover svg path {
-				fill: #2683FF;
-			}
-		}
-	}
-
-	@media screen and (max-width: 720px) {
-		.edit-profile-popup {
+    .edit-profile-row-container {
+	    width: 100%;
```
Comment (VitaliiShpital) [inline web/satellite/src/components/account/EditProfilePopup.vue]: looks like more than 4 spaces

File: web/satellite/src/components/common/HeaderedInput.vue [hunk 2]
```diff
@@ -108,65 +108,71 @@
...
+    	width: 48%;
+    }
+    
+    .label-container {
+    	display: flex;
+    	justify-content: flex-start;
+    	flex-direction: row;
+    
+    	&__add-label {
+    		margin-left: 5px;
```
Comment (VitaliiShpital) [inline web/satellite/src/components/common/HeaderedInput.vue]: looks like more than 4 spaces here

File: web/satellite/src/components/common/HeaderedInput.vue [hunk 3]
```diff
@@ -108,65 +108,71 @@
...
+    		font-family: 'font_regular';
+    		font-size: 16px;
+    		line-height: 21px;
+    		color: rgba(56, 75, 101, 0.4);
+    	}
+    
+    	&__error {
+    		color: #FF5560;
```
Comment (VitaliiShpital) [inline web/satellite/src/components/common/HeaderedInput.vue]: looks like more than 4 spaces here

File: web/satellite/src/components/common/HeaderedInput.vue [hunk 4]
```diff
@@ -108,65 +108,71 @@
...
+    
+    .optional-label-container {
+    	display: flex;
+    	flex-direction: row;
+    	justify-content: space-between;
+    	align-items: center;
+    	width: 100%;
+    
+    	h4 {
+    		font-family: 'font_regular';
```
Comment (VitaliiShpital) [inline web/satellite/src/components/common/HeaderedInput.vue]: looks like more than 4 spaces here

File: web/satellite/src/components/project/UsageReport.vue [hunk 5]
```diff
@@ -197,289 +197,309 @@
...
+                    background-color: #2683FF;
+                    
+                    p {
+                        color: #FFFFFF;
+                    }
+                    
+                    svg {
+	                    
+                        path {
+	                        fill: #ffffff !important;
```
Comment (VitaliiShpital) [inline web/satellite/src/components/project/UsageReport.vue]: looks like more than 4 spaces here

File: web/satellite/src/components/project/UsageReport.vue [hunk 6]
```diff
@@ -197,289 +197,309 @@
...
+                        
+                        &:last-child {
+                            width: 60px;
+                            
+                            p {
+                                display: none;
+                            }
+                            
+                            svg {
+	                            margin: 0;
```
Comment (VitaliiShpital) [inline web/satellite/src/components/project/UsageReport.vue]: looks like more than 4 spaces here

File: web/satellite/src/components/project/billing/BillingItem.vue [hunk 7]
```diff
@@ -44,76 +44,79 @@
...
+        margin-bottom: 10px;
+        border-radius: 6px;
+        -webkit-user-select: none;
+        -moz-user-select: none;
+        -ms-user-select: none;
+        user-select: none;
+        outline: none;
+        
+        &:hover {
+	        background: #fff;
```
Comment (VitaliiShpital) [inline web/satellite/src/components/project/billing/BillingItem.vue]: looks like more than 4 spaces here

File: web/satellite/src/components/project/billing/PaginationArea.vue [hunk 8]
```diff
@@ -31,83 +31,96 @@
...
+            border-radius: 6px;
+            width: 30px;
+            height: 30px;
+            
+            &:hover {
+                
+                svg {
+                
+                    path {
+                    fill: #fff !important;
```
Comment (VitaliiShpital) [inline web/satellite/src/components/project/billing/PaginationArea.vue]: looks like spaces are missing here

File: web/satellite/src/components/account/EditProfilePopup.vue (hunk 1 above)
Comment (crawter) [inline web/satellite/src/components/account/EditProfilePopup.vue]: extra spaces

File: web/satellite/src/components/account/EditProfilePopup.vue [hunk 9]
```diff
@@ -122,138 +122,142 @@
...
+	    display: flex;
+	    flex-direction: row;
+	    align-content: center;
+	    justify-content: flex-start;
+    }
+    
+    .edit-profile-popup-container {
+	    position: fixed;
```
Comment (crawter) [inline web/satellite/src/components/account/EditProfilePopup.vue]: extra spaces

File: web/satellite/src/components/account/billing/PaginationArea.vue [hunk 10]
```diff
@@ -31,83 +31,96 @@
...
+            align-items: center;
+            justify-content: center;
+            cursor: pointer;
+            border: 1px solid #AFB7C1;
+            border-radius: 6px;
+            width: 30px;
+            height: 30px;
+        
+            &:hover {
+
```
Comment (crawter) [inline web/satellite/src/components/account/billing/PaginationArea.vue]: do we need this empty line?

File: web/satellite/src/components/common/HeaderedInput.vue (hunk 4 above)
Comment (crawter) [inline web/satellite/src/components/common/HeaderedInput.vue]: extra spaces

File: web/satellite/src/components/project/billing/BillingItem.vue (hunk 7 above)
Comment (crawter) [inline web/satellite/src/components/project/billing/BillingItem.vue]: extra spaces

**Pull request merged**